src/interfaces/project-create-interface.ts

```diff
@@ -15,6 +15,10 @@ export interface AppConfig {
   applicationName: string;
   appHostname: string;
   appPort: number;
+  path?: string;
```

Comment:
Don't have any clue where this could come from, to be investigated

Reply:
Ok I think I got it, this has been added in the past to help integration of tests

Reply:
Final conclusion: this has been added in the past to help implementation of tests, never change your code base to ease tests, I've removed it.